docs/get-started/developers/installation.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: layout-with-subnav.njk
+title: Installation
+heading: Developers
+tags:
+  - developers
+permalink: /get-started/developers/installation/index.html
+order: 10
+bodyClasses: element-docs
+---
+
+## How to install
+
+To develop components or design system documentation, you must first install some required software, namely node.js. We use [nvm](https://github.com/nvm-sh/nvm) to ensure a uniform development environment.
+
+### Step 1: Install node
+
+**Fedora/RHEL users** should install nvm for bash directly from GitHub.
+
+<rh-code-block>
+  <script type="text/sample-javascript">
+  curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash
+  </script>
+</rh-code-block>
+
+**Mac users** should install Homebrew. Go to the website or run this command (takes 5-10 minutes).
+
+<rh-code-block>
+  <script type="text/sample-javascript">
+  /bin/bash -c "$(curl -fsSL
+  https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+  </script>
+</rh-code-block>
```

Comment:
this is technically `bash` script.   We could do something like:

```
<rh-code-block>
  <script type="text/bash">
  /bin/bash -c "$(curl -fsSL
  https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
  </script>
</rh-code-block>
```
or maybe `application/x-bsh` is the correct MIME.

FWIW the MIME used in the `type` on the script tag doesn't really do anything right now in `rh-code-block` except for the case of blocking the browser from executing the content as Javascript.  In the future it would inform what style color coding to add to the code displayed.    

For right now it's purpose of being able to use a `<script>` tag in HTML without the browser trying to run it, acknowledging it as code.  So we need to provide a type that isn't `module` | `text/javascript` | `importmap` or not existent, all those will parse and try to run the contents and no longer display as text in the  display.

Here is documentation on this:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type

> [Any other value](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type#any_other_value)
The embedded content is treated as a data block, and won't be processed by the browser. Developers must use a valid MIME type that is not a JavaScript MIME type to denote data blocks. All of the other attributes will be ignored, including the src attribute.